sky/packages/sky/lib/base/scheduler.dart

```diff
@@ -34,14 +35,19 @@ void beginFrame(double timeStamp) {
     callback(timeStamp);
 }
 
+/// Registers [beginFrame] callback with the engine.
 void init() {
   sky.view.setFrameCallback(beginFrame);
 }
 
+/// Call callback every every frame.
 void addPersistentFrameCallback(Callback callback) {
   _persistentCallbacks.add(callback);
 }
 
+/// Schedule a callback for the beginning of the next frame.
```

Comment:
describe "beginning" in more detail. "before layout"? describe how this fits in with 'rendering' and 'widgets' and their binding overrides.